tests/functional/test_provider.py

```diff
@@ -16,6 +18,15 @@ def test_get_block(eth_tester_provider, block_id):
     assert latest_block.gas_used == 0
 
 
+@mock.patch("ape.api.networks.NetworkAPI.gas_limit", new_callable=mock.PropertyMock)
```

Comment:
Note we have the `mocker` fixure from `pytest-mock` and you can do:

```python
patch = mocker.patch("ape.api.networks.NetworkAPI.gas_limit")
```

and the likes, but this is ok too, just wanted to make sure you knew it exists.

Reply:
Oh sweet, yeah I like using the fixtures more (especially avoiding any potential of multiple decorators on a test function)

Updated in e0f562aa